packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -150,9 +156,21 @@ export class PluginLedgerConnectorBesu
       `${fnTag} options.prometheusExporter`,
     );
 
+    // const web3Provider = new Web3.providers.HttpProvider(
+    //   this.options.rpcApiHttpHost,
+    // );
+
+    // this.web3 = new Web3(web3Provider);
+    this.initializeEEAClient();
```

Comment:
@travis-payne Now that we have the async plugin initializers from Tony have been merged we can leverage that for calling the EEAClient init.
Note: You'll have to call the onInitPlugin method manually on the plugin instance in the test cases (because normally that would be done by the API server)

See: https://github.com/hyperledger/cactus/blob/main/packages/cactus-core-api/src/main/typescript/plugin/i-cactus-plugin.ts#L72